app/utils/DateFormatter.scala

```diff
@@ -16,14 +16,13 @@
 
 package utils
 
-import java.time.{LocalDateTime, LocalDate => JavaDate}
-
 import com.google.inject.Inject
 import config.FrontendAppConfig
-import org.joda.time.{LocalDate => JodaDate}
 import play.api.i18n.Messages
 import uk.gov.hmrc.play.language.LanguageUtils
 
+import java.time.{LocalDateTime, LocalDate => JavaDate}
```

Comment:
now with all the JodaTime gone away, you dont need this alias you can have LocalDate everywhere and we know that is java as scala doesnt have one